doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,127 @@
+/**
+
+\page CMake-Functions FaceLift CMake API Reference
+
+\section public-functions Functions for FaceLift Users
+
+The following functions are public and are meant to be used by FaceLift users.
+
+\code
+facelift_add_qml_plugin(PluginName
+                        URI uri
+                        [VERSION version]
+                        [OUTPUT_BASE_DIRECTORY dir]
+                        [SOURCES src1 [src2...]]
+                        [HEADERS hdr1 [hdr2...]]
+                        [SOURCES_GLOB_RECURSE gsrcr1 [gsrcr2...]]
+                        [HEADERS_GLOB_RECURSE ghdrr1 [ghdrr2...]]
+                        [LINK_LIBRARIES lib1 [lib2...]])
+\endcode
+
+This function is used to build and install a Qml plugin.
+
+\param PluginName The name of the Qml plugin that will be created
+\param URI The URI of the Qml plugin
+\param VERSION The Qml plugin version, if not provided it defaults to 1.0
+\param OUTPUT_BASE_DIRECTORY The directory relative to CMAKE_BINARY_DIR, where the plugin will be installed. If not given default is "imports"
+\param SOURCES List of source files used to build the plugin
+\param HEADERS List of header files used to build the plugin
+\param SOURCES_GLOB_RECURSE List of source files used to build the plugin provided as a wildcard expression
+\param HEADERS_GLOB_RECURSE List of header files used to build the plugin provided as a wildcard expression
+\param LINK_LIBRARIES List of libraries that will be linked with the plugin
+
+<hr><br>
+
+\code
+facelift_add_interface(TargetName
+                       INTERFACE_DEFINITION_FOLDER dir
+                       [LINK_LIBRARIES lib1 [lib2...]]
+                       [IMPORT_FOLDERS folder1 [folder2...]])
+\endcode
+
+This function creates a new facelift interface library and generates the corresponding modules (like ipc, types, devtools, etc.)
+
+\param TargetName The name of the interface library that will be created
+\param INTERFACE_DEFINITION_FOLDER The folder with qface definition files
```

Comment:
mention that this folder contains the QFace files for which some code is going to be generated, unlike the "IMPORT_FOLDERS".